test/com/dotmarketing/portlets/ContentletBaseTest.java

```diff
@@ -109,8 +109,12 @@
 
     @BeforeClass
     public static void prepare () throws Exception {
-        TestingJndiDatasource.init();
-        ConfigTestHelper._setupFakeTestingContext();
+
```

Comment:
The class should have a more generic name, such as BaseIntegrationTest.
In addition the idea is to check the state of the Integration Framework, if it is not initialized, then init it. Otherwise we will reuse the current state.

It should work if you run a method, class or suite.